packages/client/hmi-client/src/composables/useCharts.ts

```diff
@@ -498,6 +499,69 @@ export function useCharts(
 		};
 	};
 
+	// Create ensemble calibrate error charts based on chart settings
+	const useEnsembleErrorCharts = (
+		chartSettings: ComputedRef<ChartSetting[]>,
+		errorData: ComputedRef<EnsembleErrorData>
+	) => {
+		const getErrorChartVariables = (configId: string) => {
+			const variables = chartSettings.value
+				.map((s) => s.selectedVariables[0])
+				.map((variable) => {
+					const modelVarName = configId
+						? getModelConfigVariable(<EnsembleVariableMappings>mapping?.value ?? [], variable, configId)
+						: variable;
+					const field = modelVarToDatasetVar(mapping?.value ?? [], variable) as string;
+					return { field, label: `${modelVarName}, ${field}` };
+				})
+				.filter((v) => !!v.field);
+			return variables;
+		};
+
+		const errorCharts = computed(() => {
+			if (!isChartReadyToBuild.value) return [];
+			const data = [errorData.value.ensemble]; // First item is always ensemble error data, and the rest are model error data
+			const modelConfigIds = extractModelConfigIdsInOrder(chartData.value?.pyciemssMap ?? {});
+			modelConfigIds.forEach((configId) => {
+				if (errorData.value[configId]) data.push(errorData.value[configId]);
+			});
+			const errorChartSpecs = data.map((ed, index) => {
+				const spec = createErrorChart(ed, {
+					title: '',
+					width: chartSize.value.width / data.length - 30, // Note: error chart adds extra 30px padding on top of the provided width so we subtract 30px to make the chart fit the container
+					variables: getErrorChartVariables(modelConfigIds[index - 1] ?? ''),
+					xAxisTitle: 'Mean absolute (MAE)',
```

Comment:
Do you mean to say `Mean absolute error (MAE)`?

Reply:
Oh yes I'll include the fix in other PR